src/test/scala/D2DDummyLoopback.scala

```diff
@@ -0,0 +1,95 @@
+package edu.berkeley.cs.ucie.digital
```

Comment:
I think this should go in the main build, in the `interfaces` package - it could conceivably be used by other consumers of this repo to test their own IP. We're putting other verification-related IP for FDI and RDI in the `interfaces` package as well.